src/pirate/psu.c

```diff
@@ -98,18 +98,24 @@ void psu_set_i(float current, struct psu_status_t* psu) {
 }
 
 void psu_dac_set(uint16_t v_dac, uint16_t i_dac) {
+#if (BP_VER != 7)
     uint slice_num = pwm_gpio_to_slice_num(PSU_PWM_VREG_ADJ);
     uint v_chan_num = pwm_gpio_to_channel(PSU_PWM_VREG_ADJ);
     uint i_chan_num = pwm_gpio_to_channel(PSU_PWM_CURRENT_ADJ);
     pwm_set_chan_level(slice_num, v_chan_num, v_dac);
     pwm_set_chan_level(slice_num, i_chan_num, i_dac);
     // printf("GPIO: %d, slice: %d, v_chan: %d, i_chan: %d",PSU_PWM_VREG_ADJ,slice_num,v_chan_num,i_chan_num);
+#endif
 }
 
 bool psu_fuse_ok(void) {
+#if (BP_VER == 7)

```

Comment:
As above, at minimum, add comment with `//BUGBUG - TODO - ` so indicate the why ... and so it can then be changed to capability-based.